prqlc/prqlc/tests/integration/snapshots/integration__queries__debug_lineage__window.snap

```diff
@@ -0,0 +1,6 @@
"parent":178},{"id":177,"kind":"Ident","span":"1:893-901","ident":{"Ident":["this","tracks","track_id"]},"targets":[127]},{"id":178,"kind":"Tuple","span":"1:820-906","children":[165,173,175],"parent":179},{"id":179,"kind":"TransformCall: Derive","span":"1:813-906","children":[127,178],"parent":181},{"id":181,"kind":"TransformCall: Take","children":[179,182],"parent":188},{"id":182,"kind":"Literal","parent":181},{"id":185,"kind":"Ident","span":"1:925-933","ident":{"Ident":["this","tracks","genre_id"]},"targets":[129],"parent":188},{"id":186,"kind":"Ident","span":"1:935-947","ident":{"Ident":["this","tracks","milliseconds"]},"targets":[127],"parent":188},{"id":188,"kind":"TransformCall: Sort","span":"1:919-948","children":[181,185,186],"parent":195},{"id":189,"kind":"Ident","span":"1:957-965","ident":{"Ident":["this","tracks","track_id"]},"targets":[127],"parent":194},{"id":190,"kind":"Ident","span":"1:967-975","ident":{"Ident":["this","tracks","genre_id"]},"targets":[129],"parent":194},{"id":191,"kind":"Ident","span":"1:977-980","ident":{"Ident":["this","num"]},"targets":[165],"parent":194},{"id":192,"kind":"Ident","span":"1:982-987","ident":{"Ident":["this","total"]},"targets":[173],"parent":194},{"id":193,"kind":"Ident","span":"1:989-997","ident":{"Ident":["this","last_val"]},"targets":[175],"parent":194},{"id":194,"kind":"Tuple","span":"1:956-998","children":[189,190,191,192,193],"parent":195},{"id":195,"kind":"TransformCall: Select","span":"1:949-998","children":[188,194],"parent":200},{"id":196,"kind":"RqOperator","span":"1:1006-1020","targets":[198,199],"parent":200},{"id":198,"kind":"Ident","span":"1:1006-1014","ident":{"Ident":["this","tracks","genre_id"]},"targets":[190]},{"id":199,"kind":"Literal","span":"1:1018-1020"},{"id":200,"kind":"TransformCall: Filter","span":"1:999-1020","children":[195,196]}],"ast":{"name":"Project","stmts":[{"VarDef":{"kind":"Main","name":"main","value":{"Pipeline":{"exprs":[{"FuncCall":{"name":{"Ident":"from"},"args":[{"Ident":"tracks"}]}},{"FuncCall":{"name":{"Ident":"group"},"args":[{"Ident":"genre_id"},{"Pipeline":{"exprs":[{"FuncCall":{"name":{"Ident":"sort"},"args":[{"Ident":"milliseconds"}]}},{"FuncCall":{"name":{"Ident":"derive"},"args":[{"Tuple":[{"FuncCall":{"name":{"Ident":"row_number"},"args":[{"Ident":"this"}]},"alias":"num"},{"FuncCall":{"name":{"Ident":"count"},"args":[{"Ident":"this"}]},"alias":"total"},{"FuncCall":{"name":{"Ident":"last"},"args":[{"Ident":"track_id"}]},"alias":"last_val"}]}]}},{"FuncCall":{"name":{"Ident":"take"},"args":[{"Literal":{"Integer":10}}]}}]}}]}},{"FuncCall":{"name":{"Ident":"sort"},"args":[{"Tuple":[{"Ident":"genre_id"},{"Ident":"milliseconds"}]}]}},{"FuncCall":{"name":{"Ident":"select"},"args":[{"Tuple":[{"Ident":"track_id"},{"Ident":"genre_id"},{"Ident":"num"},{"Ident":"total"},{"Ident":"last_val"}]}]}},{"FuncCall":{"name":{"Ident":"filter"},"args":[{"Binary":{"left":{"Ident":"genre_id"},"op":"Gte","right":{"Literal":{"Integer":22}}}}]}}]}}},"span":"1:762-1021"}]}}
```

Comment:
Possibly we should be using `to_string_pretty` so these are on multiple lines, but that's for another PR...